app/src/server/tasks/evaluateTestSetEntries.task.ts

```diff
@@ -336,12 +334,12 @@ const constructJudgementInput = (
   instructions: string | null,
 ) => {
   const input: ChatCompletionCreateParams = {
-    model: "gpt-4",
+    model: "gpt-4-0613",
     messages: [
       {
         role: "system",
         content:
-          "You are an intelligent and fair judge of chatbots. Evaluate the following two chatbot responses and choose which one is better. If the outputs are of similar quality, you can mark them as EQUAL." +
+          "You are an intelligent and fair judge of LLMs. Evaluate the following two responses and choose which one is better. If the outputs are of similar quality, you can mark them as EQUAL." +
```

Comment:
Does GPT-4-0613 have a good idea of what LLMs are?

Reply:
No, apparently doesn't. Switched this back to "chatbots".